hdrline.c

```diff
@@ -861,12 +865,15 @@ static const char *index_format_str(char *buf, size_t buflen, size_t col, int co
       if (!optional)
       {
```

Comment:
I think I'd factor out the `src[0]` check, maybe:
`const bool plain = (src[0] == 'p');`

Reply:
called it 'is_plain`